application/database/migrations/2023_04_03_122625_add_file_type_classifier_values.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use App\Enums\ClassifierValueType;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        DB::table('classifier_values')->insert(
+            array_map(function (array $classifierValueData) {
+                return [
+                    ...$classifierValueData,
+                    'type' => ClassifierValueType::FileType->value,
+                    'meta' => '[]',
+                    'created_at' => DB::raw('NOW()'),
+                ];
+            }, $this->getData())
+        );
```

Comment:
From looking at the code, it seems that the `updated_at` column is not being set. If you use Eloquent ORM to insert data, then the timestamp fields will be set automatically.

Therefore I would suggest something like this (haven’t tested): 
```suggestion
        foreach ($this->getData() as $classifierValueData) {
            ClassifierValue::create([
                ...$classifierValueData,
                'type' => ClassifierValueType::FileType->value,
                'meta' => '[]',
            ]);
        }
```